docs/tx_rollups.md

```diff
@@ -0,0 +1,108 @@
+
+## Deposits
+To be able to interact or transfer funds on layer-2, you will first need to deposit an amount to an existing layer-2 (tz4) address. 
+
+Depositing tickets from layer-1 to layer-2 can be done via a smart contract that facilitates the transfer to a tz4 address in a specified rollup node.
+
+Example of such contract can be found [here](https://tezos.gitlab.io/alpha/transaction_rollups.html?highlight=transaction%20rollups#depositing-assets-on-a-rollup)
+
+Assuming the contract have been originated, interacting with the contract in Taquito would look something like this:
+```
+const Tezos = new TezosToolkit('https://jakartanet.ecadinfra.com');
+
+const deposit = Tezos.contract.at(SMART_CONTRACT_ADDRESS);
+
+const op = await deposit.methods.default(
+    'foobar',
+    '15',
+    'tz4Jxn8MpRndqWUzkuZbQKmE3aNWJzYsSEso',
+    'txr1c9XcfmiLSWeqDPamPytns2ZXvwgYA7EpA'
+);
+
+const confirmation = await op.confirmation();
+
+console.log(op.hash);
+console.log(op.operationResults);
+```
+The `default` entrypoint takes in 4 arguments:
+- `foobar` is the ticket string
+- `15` is the quantity of a ticket string
+- `tz4Jxn8MpRndqWUzkuZbQKmE3aNWJzYsSEso` is the layer-2 address that will be the deposit recipient
+- `txr1c9XcfmiLSWeqDPamPytns2ZXvwgYA7EpA` is the rollup node id 
+
+If the deposit is successful, you will be returned an operation hash that can be accessed via `op.hash`
+
+You also might want to look at `op.operationResults` to retrieve the `ticket_hash`. A ticket hash should look something like this: `exprtz9FgfdzufUADVsvP8Gj8d8PZr9RsBEjZ5GQKM8Kp5cKWww7di`
+
+## Transfer
+Exchange of assets in the form of tickets can be done from a layer-2 address to another layer-2 address. Since transfer is a layer-2 operation, Taquito will not be supporting it. This may change in the future with SCORU (Smart Contract Optimistic Rollups).
+
+For instructions on how to conduct a transfer in layer-2 using the rollup client, refer to this [documentation](https://tezos.gitlab.io/alpha/transaction_rollups.html?highlight=transaction%20rollups#exchanging-assets-inside-a-rollup) by Nomadic Labs.
+
+## Withdrawal
+A withdrawal of assets from layer-2 back to layer-1 can be done in several steps.
+
+The first step is to perform a `withdraw` in layer-2 to a layer-1 address in the rollup client.
+```
+tezos-tx-rollup-client-alpha withdraw ${qty} of ${ticket_hash} from ${l2_src} to ${l1_dst}
+```
+
+- `${qty}` is the quantity of a ticket string that you would like to withdraw
+- `${ticket_hash}` is the ticket hash that was returned by a deposit 
+- `${l2_src}` is the BLS pair of keys generated with `tezos-client bls gen key`; or in other words, the tz4 address that holds the tickets 
+- `${l1_dst}` is the layer-1 address you would like to withdraw the tickets to
+
+After a successful withdrawal, your assets will exist back in layer-1 in the form of tickets.
```

Comment:
Should we mention the finality period? To let users know that it is not immediate.